src/config/product.rhelElsPayg.js

```diff
@@ -0,0 +1,357 @@
+            'curiosity-graph.cardBodyMetric',
+            {
+              context: ['total', dataSets?.[0]?.display?.dailyHasData && dataSets?.[0]?.display?.chartId],
+              testId: 'graphDailyTotalCard-body',
+              total: helpers
+                .numberDisplay(dataSets?.[0]?.display?.dailyValue)
+                ?.format({
+                  average: true,
+                  mantissa: 2,
+                  trimMantissa: true,
+                  lowPrecision: false
+                })
+                ?.toUpperCase()
+            },
+            [<strong title={dataSets?.[0]?.display?.dailyValue} aria-label={dataSets?.[0]?.display?.dailyValue} />]
+          ),
+        footer: ({ dataSets = [] } = {}) =>
+          translate('curiosity-graph.cardFooterMetric', {
+            date: moment
+              .utc(dataSets?.[0]?.display?.dailyDate)
+              .format(dateHelpers.timestampUTCTimeFormats.yearTimeShort),
+            testId: 'graphDailyTotalCard-footer'
+          })
+      },
+      {
+        header: ({ dataSets = [] } = {}) =>
+          translate('curiosity-graph.cardHeadingMetric', {
+            context: ['monthlyTotal', dataSets?.[0]?.display?.chartId],
+            testId: 'graphMonthlyTotalCard-header'
+          }),
+        body: ({ dataSets = [] } = {}) =>
+          translate(
+            'curiosity-graph.cardBodyMetric',
+            {
+              context: ['total', dataSets?.[0]?.display?.monthlyHasData && dataSets?.[0]?.display?.chartId],
+              testId: 'graphMonthlyTotalCard-body',
+              total: helpers
+                .numberDisplay(dataSets?.[0]?.display?.monthlyValue)
+                ?.format({
+                  average: true,
+                  mantissa: 2,
+                  trimMantissa: true,
+                  lowPrecision: false
+                })
+                ?.toUpperCase()
+            },
+            [<strong title={dataSets?.[0]?.display?.monthlyValue} aria-label={dataSets?.[0]?.display?.monthlyValue} />]
+          ),
+        footer: ({ dataSets = [] } = {}) =>
+          translate('curiosity-graph.cardFooterMetric', {
+            date: moment
+              .utc(dataSets?.[0]?.display?.monthlyDate)
+              .format(dateHelpers.timestampUTCTimeFormats.yearTimeShort),
+            testId: 'graphMonthlyTotalCard-footer'
+          })
+      }
+    ],
+    isCardTitleDescription: true,
+    xAxisChartLabel: () => translate('curiosity-graph.label_axisX', { context: GRANULARITY_TYPES.DAILY }),
+    yAxisTickFormat: ({ tick } = {}) => {
+      if (tick > 1) {
+        return helpers
+          .numberDisplay(tick)
+          ?.format({ average: true, mantissa: 1, trimMantissa: true, lowPrecision: false })
+          ?.toUpperCase();
+      }
+      return helpers
+        .numberDisplay(tick)
+        ?.format({ average: true, mantissa: 5, trimMantissa: true, lowPrecision: true })
+        ?.toUpperCase();
+    }
+  },
+  initialInventoryFilters: [
```

Comment:
inventory configuration should focus on single display metric, use RHACS as the ref